main.py

```diff
@@ -1,16 +1,147 @@
+import base64
+import datetime
+import glob
+import json
 import os
+import ssl
+import subprocess
+
+import certifi
 import decky_plugin
+from aiohttp import ClientSession
 from settings import SettingsManager
+from typing_extensions import Optional
+
 
 class Plugin:
+    yt_process: Optional[subprocess.Popen] = None
```

Comment:
Using `Optional` and importing `typing_extensions` shouldn't be needed here. The backend runs on Python 3.11, we can instead use `subprocess.Popen | None` or `asyncio.subprocess.Process | None` instead instead.

Reply:
Right, fixed in 49d32b6.